clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3086,17 +3086,28 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
       return;
     ZDimExpr = ZDim.get();
 
+    // If the declaration has a [[intel::num_simd_work_items()]] attribute,
+    // check to see if the first argument of
+    // __attribute__((reqd_work_group_size)) attribute on OpenCL mode and
+    // last argument of [[cl::reqd_work_group_size()]] or
+    // [intel::reqd_work_group_size()]] attribute or
+    // __attribute__((reqd_work_group_size)) attribute on SYCL mode can be
+    // evenly divided by the [[intel::num_simd_work_items()]] attribute.
```

Comment:
This comment is slightly off -- it's no longer a matter of spelling but language options.